Distributions/OpacitySource/wlOpacityFieldsModule.f90

```diff
@@ -262,20 +275,21 @@ SUBROUTINE DescribeOpacityTypeScat( Opacity )
 
       DO l = 1, Opacity % nMoments
       WRITE(*,*)
-         IF( Opacity % nMoments .eq. 4 )THEN
+         !IF( Opacity % nMoments .eq. 4 )THEN
+         IF( size(Opacity % Kernel(i) % Values, DIM=1) /= size(Opacity % Kernel(i) % Values, DIM=2) ) THEN
```

Comment:
What does this line do?

Reply:
the hardcoded checking for Opacity % Moments .eq. 4 to distinguish between 
' ', 'Min Value = ', MINVAL( Opacity % Kernel(i) % Values(:,l,:,:,:) )
and 
' ', 'Min Value = ', MINVAL( Opacity % Kernel(i) % Values(:,:,l,:,:) )
doesn't work anymore with Brem being in there, so I checked for a different criterion